src/apps/tools/OswAppCalculator.cpp

```diff
@@ -0,0 +1,448 @@
+/*
+    a calculator for addition, subtraction, multiplication, division, square roots, powers, num1 * 10 ^ num2
+    with postitive and negativ numebers + generating random numbers
+    press btn2 + btn3 to set the cursor a position back
+*/
+
+#include "./apps/tools/OswAppCalculator.h"
+
+#include "config_defaults.h"  // Include the config here again to access the language definitions of it
+#include "gfx_util.h"
+#include "osw_app.h"
+#include "osw_hal.h"
+#include "osw_ui.h"
+
+#include <osw_config_keys.h>
+
+char pNum1 = '+';
+int num1[] = {0, 0, 0, 0, 0, 0};
+double Num1;
+String op = "+";
+char pNum2 = '+';
+int num2[] = {0, 0, 0, 0, 0, 0};
+double Num2;
+int8_t i = 0;
```

Comment:
What is mean `i`? 
Please modify the meaning of the variable to make it easier to understand. 
E.g `count`, `position`.